### ISSUE TITLE
[Feature Request] Add more styling options to VExpansionPanels to make it more like VCard

### ISSUE THREAD

**Issue by MatthewAry** [vuetifyjs/vuetify#12039]
### Problem to solve
VExpansionPanels when collapsed, has the visual appearance of a VCard with content inside of it. However the styling capabilities of VExpansionPanels are more limited than VCard. I would like to request a change to address this. I understand that this is would mean a major rewrite, but V3 is coming up so I think it's a good time to make this kind of proposal.

### Proposed solution
If possible, it would be great if VExpansionPanels could obtain the following functionality (by way of props) that is enjoyed by VCard.
- color
- elevation
- height
- img
- loader-height
- loading
- max-height
- max-width
- min-height
- min-width
- outlined
- raised
- rounded
- shaped
- width

**Comment by douglasg14b** [comment]
Yes, please. Ideally panel-like components should be style-able in the same manners. Right now styling an expansion panel is an exercise in tedium.
